code/game/machinery/pipe/construction.dm

```diff
@@ -20,6 +20,8 @@ Buildable meters
 	w_class = WEIGHT_CLASS_NORMAL
 	var/piping_layer = PIPING_LAYER_DEFAULT
 	var/RPD_type
+	/// Whether it can be painted
+	var/paintable = FALSE
```

Comment:
Please mention changes like this in the pr body in future, it makes discerning intent a lot easier

Reply:
sure